src/lib/network_pool/network_pool_base.ml

```diff
@@ -43,10 +44,8 @@ end)
     |> ignore ;
     network_pool
 
-  let create ~logger ~pids ~trust_system ~incoming_diffs
-      ~frontier_broadcast_pipe =
+  let create ~config ~incoming_diffs ~frontier_broadcast_pipe ~logger =
```

Comment:
Do we need this logger? It seems like all the configs already contain a logger

Reply:
Because the type of `config` is abstract here and `Network_pool.t` needs a logger. I could leave logger out of `Config.t` and pass it explicitly everywhere?